app/assets/stylesheets/users.scss

```diff
@@ -4,12 +4,12 @@
 input[type="text"] {
     height: 27px;
     margin-bottom: 15px;
-    border-radius: 8px;
+    border-radius: 0px;
 }
 input[type="password"] {
     height: 27px;
     margin-bottom: 15px;
-    border-radius: 8px;
+    border-radius: 0px;
```

Comment:
I don't like the rounded input box inconsistency, so either revert or do the rest.